GoogleChromeLabs/ps-analysis-tool
# Fix: GSI messaging on correct urls.

## Description

Sometimes when the site is reloaded quickly then the GSI shows error saying we are on invalid URL. This PR aims to fix that.

## Relevant Technical Choices



## Testing Instructions
- Clone this branch and in terminal run `npm start`.
- Open a fresh browser instance.
- Go to linkedin.com
- Open PSAT.
- Now click on reload button twice (Start a reload cancel the reload and then start again).
- GSI section should not show invalid URL message.

## Additional Information:



## Screenshot/Screencast



---

## Checklist






Fixes #681
